packages/client/katello-pull-provider-migrate/katello-pull-provider-migrate.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/sh
+
+# check if system is registered
+subscription-manager identity > /dev/null
+if [ $? -ne 0 ]; then
+    echo "Please register with subscription-manager first, then re-run `katello-pull-provider-migrate.sh`."
+    exit 1
+fi
+
+echo "Getting configuration from subscription-manager..."
+KPPTEMPFILE=/tmp/kpp_tempfile
```

Comment:
```suggestion
KPPTEMPFILE=$(mktemp)
```

Otherwise you allow attackers to overwrite arbitrary files by pre-creating `/tmp/kpp_tempfile`

Reply:
PR reviews are the best Linux crash course :smile: